pylama.ini

```diff
@@ -0,0 +1,16 @@
+[pylama]
+linters = pycodestyle,pydocstyle,pyflakes,pylint
+ignore = D203,D213,D406,D407,C0103,W0511
```

Comment:
I think that the pip install command automatically adds the following linters as well, mccabe and isort. You might want to add these here.